subsys/net/lib/fota_download/Kconfig

```diff
@@ -37,6 +37,17 @@ config FOTA_DOWNLOAD_NATIVE_TLS
 	  Enabling this option will configure the socket to be native for TLS
 	  instead of offloading TLS operations to the modem.
 
+config FOTA_DOWNLOAD_FILE_NAME_LENGHT
+	int "Size of buffer used for file name"
+	default 128
+
+config FOTA_DOWNLOAD_HOST_NAME_LENGHT
+	int "Size of buffer used for host name"
+	default 128
+config FOTA_DOWNLOAD_RESOURCE_LOCATOR_LEN
+	int "Size of buffer used for Resource locator"
+	default 500
```

Comment:
why not 512? Power of two is always best

Reply:
This was already 500 before I move define to kConfig. I'm fine for 512